lib/Doctrine/ORM/Query/Parser.php

```diff
@@ -1280,7 +1284,9 @@ public function DeleteClause()
             $this->match(Lexer::T_AS);
         }
 
-        $aliasIdentificationVariable = $this->AliasIdentificationVariable();
+        $aliasIdentificationVariable = $this->lexer->isNextToken(Lexer::T_IDENTIFIER)
+            ? $this->AliasIdentificationVariable()
+            : 'alias_should_have_been_set';
```

Comment:
We should likely raise a deprecation notice here, to clarify that this isn't supported DQL (according to BNF)

Reply:
Didn't we say that deprecation notices were going to be added in `v2.7` only?

Reply:
Works for me as long as you please create an issue tracking this specific deprecation 👍

Reply:
Sure, I was going to do that anyway 😄